docs/docs/migration/v0.22.2.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 0
+title: v0.22.2
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.22.2, changes are required to use Ignite CLI v0.22.2. 
+---
+
+# Upgrading a blockchain to use Ignite CLI v0.22.2
+
+Ignite CLI v0.22.2 changed the GitHub username from "ignite-hq" to "ignite".
+
+1. Open your `go.mod` and change the Ignite CLI line with `github.com/ignite/cli v0.22.2`
+
+2. In your `go.sum` file change `github.com/ignite-hq/cli` to `github.com/ignite/cli`
+
+3. Make sure that your all CLI import lines in yout app start with `github.com/ignite/cli`, including these files:
+
+ - `app/app.go`
+ - `app/simulation_test.go`
+ - `testutil/network/network.go`
+
+4. Finally open the `main.go` file located inside the app's `cmd/` directory and also make sure the cosmoscmd package is imported as `github.com/ignite/cli/ignite/pkg/cosmoscmd`
```

Comment:
```suggestion
```

Should be covered by step 3.

Reply:
This one is trickier to make it precise because the app name can be anything. I will see if I figure out a good way to document it.